### PR TITLE
Added force option to deploy stacksets without intervention

### PR DESCRIPTION
**Pull request by hariprakash-j**
*Description of changes:*

Added `-f` or `--force` to deploy stacksets without user intervention and generated the updated docs.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by hariprakash-j** [comment]
A bit more context as to why this is needed, I use this in a CICD pipeline and not having this option makes it unusable for stacksets.

File: docs/rain_stackset_deploy.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ rain stackset deploy <template> [stackset] [flags]
       --accounts strings   accounts for which to create stack set instances
   -c, --config string      YAML or JSON file to set additional configuration parameters
   -d, --detach             once deployment has started, don't wait around for it to finish
+  -f, --force              update the stackset without confirmation
```
Comment (ericzbeard) [inline docs/rain_stackset_deploy.md:46]: Would -y be a better option to match the deploy command?

**Review by ericzbeard — CHANGES_REQUESTED** [review]

File: docs/rain_stackset_deploy.md (hunk 1 above)
Comment (hariprakash-j) [reply docs/rain_stackset_deploy.md]: yup that's more consistent. I fixed it.

**Review by ericzbeard — COMMENTED** [review]
Looks good but we have conflicts with the auto generated docs from another PR.

**Comment by hariprakash-j** [comment]
fixed the conflicts :+1:

**Review by ericzbeard — APPROVED** [review]

**Pull request merged**